pkg/cmd/orderbook.go

```diff
@@ -62,9 +66,76 @@ var orderbookCmd = &cobra.Command{
 	},
 }
 
+// go run ./cmd/bbgo orderupdate
```

Comment:
do we really need this command? looks like only for testing and development

Reply:
Yes it's for testing only. Do I have to remove them?

Reply:
move to examples?